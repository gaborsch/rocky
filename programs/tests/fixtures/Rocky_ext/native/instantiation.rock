from java, math play BigDecimal, BigInteger

X will be BigDecimal taking "1.0"
Y will be BigDecimal taking 2
say "Instantiation OK"

say "X is " + X
say "Y is " + Y
