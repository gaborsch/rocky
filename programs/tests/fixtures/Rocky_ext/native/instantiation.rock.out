Instantiation OK
X is 1.0
Y is 2